server/src/test/java/io/crate/integrationtests/GeoShapeIntegrationTest.java

```diff
@@ -89,6 +95,16 @@ public void testSelectGeoShapeFromSource() throws Exception {
         assertThat(response.rows()[0][0], instanceOf(List.class));
     }
 
+    @Test
+    public void testSelectGeoShapeFromSource_bkdTree() {
```

Comment:
Minor: let's use underscore for the whole test name (and in other similar changes).